### ISSUE TITLE
Can you open the Ghidra website to Israeli IP addresses?

### ISSUE THREAD

**Issue by assafnativ** [NationalSecurityAgency/ghidra#1095]
https://ghidra-sre.org/ is blocked for Israeli IP addresses. Israel and the US are BFF, and this behavior is very not BFF like.

**Comment by ryanmkurtz** [comment]
Duplicate of #32

**Comment by ryanmkurtz** [comment]
Sorry for the inconvenience...we are looking into it.

**Comment by assafnativ** [comment]
Sorry for the duplicate. There are too many issues open...

**Comment by dukase** [comment]
Asaf, Just use VPN and download it.

**Comment by assafnativ** [comment]
@dukase, you don't even need a VPN, a free in-browser proxy service will do the job just fine. It's still quite annoying.

**Comment by faissaloo** [comment]
Consider moving.

**Comment by endrin** [comment]
> Consider moving.
No, consider building from master. It's not that hard to set up gradle, builds rarely break, and it gets some frequently reported issues ~~yes, I'm referring to PDB ones~~ being fixed pretty fast.

**Comment by dukase** [comment]
Still not working for Israeli IP

**Comment by kadrunesia** [comment]
Blocked by NSA or your internet provider? I think NSA doesn't need to do that because they put source at Github.

**Comment by ryanmkurtz** [comment]
The Ghidra team has no control over this aspect of the website, so this issue is considered out of scope for the goals of this GitHub project (and its associated open/closed Issues).   However, we've passed this connectivity issue along to the correct people and it is up to them to change the behavior of the server/networking rules at this point.  Since that is as much of a "fix" as we can do, we consider this issue closed.  

Rather than continuing to add to this closed issue (or open new ones), I would recommend adding your support to #1056 since that is something we do have control over.
